v2/manager.go

```diff
@@ -560,6 +560,23 @@ func (c *Manager) freeze(path string, state State) error {
 	}
 }
 
+func (c *Manager) isCgroupEmpty() bool {
+	// In case of any error we return true so that we exit and don't leak resources
+	out := make(map[string]interface{})
+	err := readKVStatsFile(c.path, "cgroup.events", out)
+	if err != nil {
+		return true
+	}
+	if v, ok := out["populated"]; ok {
+		populated, ok := v.(uint64)
+		if !ok {
+			return true
+		}
+		return populated == 0
```

Comment:
This changes the behavior of `EventChan()` if I read code correctly. In the following scenario:
1. Create cgroup Z.
2. Move process A to cgroup Z.
3. Terminate process A.
4. Move process B to cgroup Z.

After step 3. the watching loop will exit and be cleaned up, which is currently not the case. Is that desired?

Reply:
Make sense, thanks for explanation!